## 11_gcc15.diff by <hille42@debian.org>
##
## Replace hand written bool definition by #include .
## Bug#1096437

--- cjk.orig/utils/hbf2gf/hbf.c
+++ cjk/utils/hbf2gf/hbf.c
@@ -62,6 +62,7 @@
 #include <stdio.h>
 #include <ctype.h>
 #include <string.h>
+#include <stdbool.h>
 #include "hbf.h"
 
 #ifdef __MSDOS__
@@ -82,7 +83,7 @@
 
 #define	reg	register
 
-typedef	int	bool;
+/* typedef	int	bool; */
 #define	TRUE	1
 #define	FALSE	0
 
